index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <nav class="nav-bar">
+      <a href="/" class="logo">My Logo</a>
+      <img class="menu" src="/img/Union.svg " alt="" />
+    </nav>
+    <section class="headline">
+      <div class="container">
+        <h1>I’m Mohammed Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+      </div>
+      <div class="contact-me">
+        <p>LET'S CONNECT</p>
+        <div>
+          <a href="https://twitter.com/RebbouhElseddik"
+            ><img src="img/twitter.svg" alt=""
+          /></a>
+          <a href=""><img src="img/Vector(1).svg" alt="" /></a>
+          <a href=""><img src="img/Vector(2).svg" alt="" /></a>
+          <a href="https://github.com/MohammedElseddik">
+            <img src="img/github.svg" alt=""
+          /></a>
+          <a href=""><img src="img/Vector(4).svg" alt="" /></a>
+        </div>
```

Comment:
- [ ] Please consider using a `ul` tag to wrap all the social icons instead of `div` tag and use `li` tag to wrap each of the social icons. This is because the social icons are unordered lists and therefore should be wrapped with a `ul` and `li` tag and also to improve the code structure.